src/transformState.js

```diff
@@ -5,7 +5,27 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (let i = 0; i < actions.length; i++) {
+    const object = actions[i];
+
+    if (object.type === 'addProperties') {
```

Comment:
Move all cases to the object.

Example of usage: `object.type === ActionTypes.AddProperties`